.rubocop.yml

```diff
@@ -136,7 +96,8 @@ Lint/HandleExceptions:
     - 'spec/jobs/generate_flat_post_job_spec.rb'
 
 Lint/ParenthesesAsGroupedExpression:
-  Enabled: false
+  Exclude:
+    - 'spec/**/*'
```

Comment:
How many issues do we get if we don't exclude this? I'm not sure where we'd be breaching it here, and we probably shouldn't breach it.

Reply:
I think it was mis-firing (at least I _think_ that was mis-firing) on some expectation stuff. So... a lot.